CMakeLists.txt

```diff
@@ -1,7 +1,10 @@
 cmake_minimum_required(VERSION 3.16)
 project(ArborX CXX)
 
-find_package(Kokkos 3.7.01 REQUIRED CONFIG)
+find_package(Kokkos QUIET 3.7.01 CONFIG)
+if (NOT Kokkos_FOUND)
```

Comment:
Please add a comment that this is a workaround for building against the last version of Trilinos x.y.z that does not set the right compatibility mode for Kokkos